subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/TestTaskIntegrationTest.groovy

```diff
@@ -282,7 +282,7 @@ class TestTaskIntegrationTest extends JUnitMultiVersionIntegrationSpec {
         extraArgs << [[], ["--tests", "MyTest"]]
     }
 
-    def "options set prior to setting same test framework will warn and have no effect"() {
+    def "options set prior to setting same test framework will remain in effect and not cause error"() {
```

Comment:
This doesn't seem to describe the behavior demonstrated in the test, does it?

Reply:
Thanks for pointing this out.  I was experimenting with making that situation not an error and missed reverting this when abandoning that approach.